apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -253,6 +253,12 @@ type EnvoyConfig struct {
 	// +optional
 	Service *NamespacedName `json:"service,omitempty"`
 
+	// LoadBalancer holds load balancer parameters for Ingress status.
+	//
+	// Contour's default is "service:projectcontour/envoy"
+	// +optional
+	LoadBalancer string `json:"loadBalancer,omitempty"`
```

Comment:
For the documentation, it would be great to include examples here on the supported "kinds", to give overall idea what is possible to configure. This comment will be basis of generated API docs. Besides the generated API docs, the new option should be documented also in [`site/content/docs/main/configuration.md`](https://github.com/projectcontour/contour/blob/main/site/content/docs/main/configuration.md).

Reply:
Thanks for calling this out! I've updated both.